scripts/fiber/tests-failing.txt

```diff
@@ -110,6 +110,7 @@ src/renderers/shared/shared/__tests__/ReactStatelessComponent-test.js
 
 src/renderers/shared/shared/__tests__/ReactUpdates-test.js
 * should queue mount-ready handlers across different roots
+* should flush updates in the correct order
```

Comment:
Ah the previous, failing test is putting the scheduler into an inconsistent state, causing this test to fail. Something isn't being reset in a `finally` block. I had a similar bug earlier, should be easy to narrow down and fix.